modules/op-release-notes-1-2.adoc

```diff
@@ -94,6 +102,21 @@ oc patch el/github-listener-interceptor -p '{"metadata":{"finalizers":["foregrou
 oc patch crd/eventlisteners.triggers.tekton.dev -p '{"metadata":{"finalizers":[]}}' --type=merge
 ----
 
+* When running a multi-arch container image Task without command specification on an IBM Power Systems (ppc64le) or IBM Z (s390x) cluster, Task run fails with the following error:
```

Comment:
- Suggest: When you run...
- As per the [latest guidelines on OCP](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references): s/ Task/`Task` resource
- ...without a command ...
- IBM Power Systems/IBM Power System ?
- ...cluster, the `TaskRun` resource fails...